arkouda/dtypes.py

```diff
@@ -63,13 +63,13 @@ def translate_np_dtype(dt: np.dtype) -> Tuple[builtins.str, int]:
     kind = trans[dt.kind]
     return kind, dt.itemsize
 
-def resolve_scalar_dtype(val : object) -> str:
+def resolve_scalar_dtype(val : Union[np.bool,np.float64,int,np.int,np.uint8]) -> str: #type: ignore
     """
     Try to infer what dtype arkouda_server should treat val as.
     """
     # Python bool or np.bool
     if isinstance(val, builtins.bool) or (hasattr(val, 'dtype') \
-                                and val.dtype.kind == 'b'):
+                                and cast(np,val).dtype.kind == 'b'):
```

Comment:
What does this cast do? I can't figure it out...

Reply:
@reuster986 bwoops, not much, will fix, thanks for catching that!

Reply:
@reuster986 fixed